extensions/rates/functions/src/config.ts

```diff
@@ -1,9 +1,12 @@
+const splitAndTrimString = (value: string): string[] =>
+  value.split(',').map((id) => id.trim());
+
 export default {
   location: process.env.LOCATION!,
   databaseInstance: process.env.SELECTED_DATABASE_INSTANCE!,
   shipEngineApiKey: process.env.SHIPENGINE_API_KEY!,
   collectionPath: process.env.COLLECTION_PATH!,
-  carrierIds: process.env.CARRIER_IDS!.split(',').map((id) => id.trim()),
+  carrierIds: splitAndTrimString(process.env.CARRIER_IDS!),
```

Comment:
TBH I preferred the way this was previously. Unnecessary abstraction until we actually have a need to split and trim strings multiple places in this file

Reply:
yeah I saw that coming. It made me feel good. 👍

Reply:
updated!